crates/communication/src/server/receiver.rs

```diff
@@ -104,7 +115,18 @@ async fn handle_message(
                         .expect("receiver should always wait for all senders");
                 }
                 Request::Injections(_) => todo!(),
-                Request::Parameters(_) => todo!(),
+                Request::Parameters(request) => {
+                    parameters_sender
+                        .send(parameters::ClientRequest {
+                            request,
+                            client: Client {
+                                id: client_id,
+                                response_sender: response_sender.clone(),
+                            },
```

Comment:
This could be constructed outside the match statement.

Reply:
Done